plugin/src/main/java/io/fabric8/maven/plugin/mojo/infra/AbstractInstallMojo.java

```diff
@@ -282,16 +309,16 @@ protected Platform getPlatform() {
         }
     }
 
-    protected void runGofabric8(File command, String ... args) throws MojoExecutionException {
+    protected void executeCommand(File command, String binName, String ... args) throws MojoExecutionException {
         // Be sure to run in batch mode
         List<String> argList = new ArrayList<>(Arrays.asList(args));
-        argList.add("--batch");
+        //argList.add("--batch");
```

Comment:
I see, I have commented from here but added it to `executeCommand(gofabric8, GOFABRIC8,"version", "--batch");` command line parameter for GOFABRIC8, while kompose does need this parameter hence to reuse the method I have made small change